internal/inbox/inbox.go

```
@@ -486,6 +486,21 @@ func (m *Manager) UpdateConfig(id int, config json.RawMessage) error {
 }
 
+// CloseLiveChatClients disconnects widget websocket clients and returns the number of inboxes closed.
+func (m *Manager) CloseLiveChatClients() int {
+	m.mu.RLock()
+	defer m.mu.RUnlock()
+	var n int
+	for _, inb := range m.inboxes {
+		if inb.Channel() != ChannelLiveChat {
+			continue
+		}
+		inb.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `inb.Close` is not checked

(errcheck)